package.json

```diff
@@ -45,15 +47,24 @@
   },
   "dependencies": {
     "@azure/cosmos": "^3.7.2",
+    "@babel/runtime": "^7.9.0",
     "applicationinsights": "^1.7.3",
     "azure-storage": "^2.10.3",
     "cidr-matcher": "^2.1.0",
     "fp-ts": "1.17.4",
+    "gulp": "^3.9.1",
+    "gulp-json-editor": "^2.5.4",
+    "gulp-prettier": "^3.0.0",
+    "gulp-rename": "^2.0.0",
+    "gulp-run": "^1.7.1",
```

Comment:
do we need this dep? (can you please review the added packages?)

Reply:
Nope this is useless (I've copied it from io-functions). I'll remove it.